example/postgres-image/Dockerfile

```diff
@@ -1,3 +1,5 @@
 FROM postgres:10.0
 COPY ./airports.csv /
```

Comment:
Remove `airports.csv` now if new testdata is being used?

Reply:
I will keep both one the make sure that the correct types gets loaded and the other one to simulate a "bulk" test.